src/app/app.html

```diff
@@ -9,6 +9,10 @@
     <ion-content>
       <ion-list role="navigation" id="navigation-menu" aria-label="Peeveetrack Menu">
         <button role="link" menuClose ion-item *ngFor="let p of pages" (click)="openPage(p)">
+        <ion-icon name="ios-heart-outline" *ngIf="p.title == 'Favorites'"></ion-icon>
```

Comment:
Okay. I'll make it so the icon name is a variable. And I'll go from there in terms of abstracting information. Idk if that's optimal.